client/src/services/userService.js

```diff
@@ -1,11 +1,12 @@
+import axios from 'axios';
 import cookies from 'js-cookie';
+import { helpers } from '../common/helpers';
 
 class UserService {
   static authorizeUser() {
-    // ToDo: ReEvaluate placement of this spoof for auth. Also consider using a helper function.
-    if (process.env.REACT_APP_ENV === 'development') {
+    // ToDo: ReEvaluate placement of this spoof for auth
+    if (helpers.DEV_MODE) {
```

Comment:
I'm having trouble finding this in helpers?

Reply:
bah... also coming soon. Multi-tasking.... will fix.